lib/matplotlib/offsetbox.py

```diff
@@ -263,32 +263,22 @@ def set_offset(self, xy):
         xy : (float, float) or callable
             The (x, y) coordinates of the offset in display units. These can
             either be given explicitly as a tuple (x, y), or by providing a
-            function that converts the extent into the offset. This function
-            must have the signature::
-
-                def offset(width, height, xdescent, ydescent, renderer) \
--> (float, float)
+            function dynamically computes an offset (taking the arguments
+            passed to `.OffsetBox.get_offset`).  It is recommended, but not
+            compulsory, to make such functions take no arguments.
         """
         self._offset = xy
         self.stale = True
 
-    def get_offset(self, width, height, xdescent, ydescent, renderer):
+    def get_offset(self, *args, **kwargs):
         """
-        Return the offset as a tuple (x, y).
-
-        The extent parameters have to be provided to handle the case where the
-        offset is dynamically determined by a callable (see
-        `~.OffsetBox.set_offset`).
-
-        Parameters
-        ----------
-        width, height, xdescent, ydescent
-            Extent parameters.
-        renderer : `.RendererBase` subclass
+        Return the (x, y) offset.
 
+        Parameters must be passed if the offset is dynamically determined by a
+        callable (see `~.OffsetBox.set_offset`), and are forwarded to that
+        callable.
```

Comment:
```suggestion
        Parameters are usually not necessary. The only exception can occur
        if you have defined a callable to calculate the offset dynamically (see
        `~.OffsetBox.set_offset`). It is now recommended that such a
        callable does not take parameters. However, for backward-compatibility
        we still support callables with parameters, and these parameters must
        be provided to `.get_offset` so that we can pass them on.
```